spanner/integration_test.go

```diff
@@ -146,12 +156,28 @@ var (
 		`CREATE INDEX TestTableByValueDesc ON TestTable(StringValue DESC)`,
 	}
 
+	readDBPGStatements = []string{
+		`CREATE TABLE TestTable (
+                    Key          VARCHAR PRIMARY KEY,
+                    StringValue  VARCHAR
+            )`,
```

Comment:
the indentation looks a big weird here. Have we run any linter, like goimports, to format the code?